[CI] MultiClusterSpecIT test {string.EndsWithLucenePushdownIgnoreMultivalues} failing

**Build Scans:**
- [elasticsearch-periodic #6983 / 8.13.4_bwc](https://gradle-enterprise.elastic.co/s/c5sgf3bgfys3e)
- [elasticsearch-periodic #6976 / 8.13.4_bwc](https://gradle-enterprise.elastic.co/s/4pll5d226jihw)
- [elasticsearch-periodic #6962 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/citf72gmh2b4e)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-clusters:v8.13.4#oldToNew" -Dtests.class="org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT" -Dtests.method="test {string.EndsWithLucenePushdownIgnoreMultivalues}" -Dtests.seed=AFFCF24224436059 -Dtests.bwc=true -Dtests.locale=lu-CD -Dtests.timezone=America/Whitehorse -Druntime.java=24
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Bstring.EndsWithLucenePushdownIgnoreMultivalues%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Unexpected warning: Line -1:-1: evaluation of [] failed, treating result as null. Only first 20 failures recorded.
```

**Issue Reasons:**
- [8.x] 3 failures in test test {string.EndsWithLucenePushdownIgnoreMultivalues} (0.3% fail rate in 998 executions)
- [8.x] 2 failures in step 8.13.4_bwc (66.7% fail rate in 3 executions)
- [8.x] 3 failures in pipeline elasticsearch-periodic (75.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec
@@ -981,6 +981,7 @@ beta         | Kubernetes cluster
 ;
 
 startsWithLucenePushdownIgnoreMultivalues
+required_capability: starts_with_ends_with_lucene_pushdown
 
 from hosts
 | where starts_with(description, "epsilon")
@@ -1266,6 +1267,7 @@ beta         | Kubernetes cluster
 ;
 
 endsWithLucenePushdownIgnoreMultivalues
+required_capability: starts_with_ends_with_lucene_pushdown
 
 from hosts
 | where ends_with(description, "host")

--- a/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java
+++ b/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java
@@ -873,6 +873,12 @@ public class EsqlCapabilities {
         RRF(Build.current().isSnapshot()),
 
         /**
+         * Lucene query pushdown to StartsWith and EndsWith functions.
+         * This capability was created to avoid receiving wrong warnings from old nodes in mixed clusters
+         */
+        STARTS_WITH_ENDS_WITH_LUCENE_PUSHDOWN,
+
+        /**
          * Full text functions can be scored when being part of a disjunction
          */
         FULL_TEXT_FUNCTIONS_DISJUNCTIONS_SCORE;

